### PR TITLE
Documentation fixes 

### PR DESCRIPTION
**Pull request by mikee47**
@kmihaylov Re #2106 I've included a basic README for the CS5460 library, if you'd like to add anything you could do it here!

### CODE REVIEW

**Comment by mikee47** [comment]
You can browse the resulting documentation for this PR at https://smingdev.readthedocs.io/en/fix-documentation/

**Comment by slaff** [comment]
@mikee47 Ping me when this PR is ready.

**Comment by mikee47** [comment]
@slaff All done, thanks.

File: docs/source/framework/core/data/streams/index.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Streams
+=======
+
+Sming provides a set of Stream class which extend :cpp:class:`Stream` methods.
+
+:cpp:class:`IDataSourceStream` is used where read-only access is required.
+It introduces the :cpp:func:`IDataSourceStream::readMemoryBlock` method which
+performs a regular read without updating the stream position.
+This allows optimistic reading and re-sending, but cannot be handle by some stream
```
Comment (slaff) [inline docs/source/framework/core/data/streams/index.rst:9]: > but cannot be handle by some stream

Did you mean "but cannot be handle**d** by some stream"

**Comment by slaff** [comment]
Ready?

**Comment by mikee47** [comment]
Yep. Hit it!

**Pull request merged**